pages/project/read/views/index.jsx

```diff
@@ -74,7 +74,7 @@ export default function ProjectLandingPage() {
     delete sections.history;
   }
 
-  if (project.status === 'inactive') {
+  if (project.status === 'inactive' || !asruLicensing) {
```

Comment:
I think this needs a few more conditions to cover AA and the like also. I think on L71 also get `canUpdate` and add a `|| !canUpdate` here too.

Reply:
I know that will be a no-op _right now_, but it means when we do the story to not lock it down to licensing only then we won't need to add anything new so there'll be no risk of exposing it too widely.